File: modul 4/python/PRAK405-2410817110008-MuhammadAzmaAlFaqih.py
n, num = map(int, input().split())
sum = 0

for i in range(1,n+1):
    result = 0
    for j in range(i,0, -1):
        print(f"({j} * {num})", end=" ")
        sum += j * num
        result += j * num
    print(f"= {result}")
print(sum)

File: modul 4/c/PRAK405_2410817110008_MuhammadAzmaAlFaqih.c
#include <stdio.h>

int main(){
  int n, num;
  int sum = 0 , result = 0;
  scanf("%d %d", &n, &num);

  for(int i = 1;i<=n;i++){
    result = 0;
    for(int j = i;j>0;j--){
      printf("(%d * %d)", j, num);
      (j>1) ? printf(" + ") : printf(" = ");
      result += j*num;
      sum += j*num;
    }
    printf("%d\n", result);
  }
  printf("%d", sum);
}
